src/Compilers/CSharp/Portable/Lowering/LambdaRewriter/LambdaRewriter.Analysis.cs

```diff
@@ -374,6 +289,62 @@ void RecordClosureScope(Scope innermost, Scope outermost, Closure closure)
             }
 
             /// <summary>
+            /// Walk up the scope tree looking for a closure.
+            /// </summary>
+            /// <returns>
+            /// A tuple of the found <see cref="Closure"/> and the <see cref="Scope"/> it was found in.
+            /// </returns>
+            public static (Closure, Scope) FindClosureInScope(Scope startingScope, MethodSymbol closureSymbol)
```

Comment:
I was trying to find some way to say "this method searches up", as opposed to the other one, which searches down. Any suggestions?